Snowflake.Data/Core/Session/SessionPool.cs

```diff
@@ -485,5 +533,29 @@ public bool SetPooling(bool isEnable)
         }
 
         public bool GetPooling() => _poolConfig.PoolingEnabled;
+
+        internal int OngoingSessionCreationsCount()
```

Comment:
i'm assuming that's a follow-up work, right?

Reply:
no, it is a method used only in tests. It is used in waiting conditions.